core/src/test/scala/com/microsoft/ml/spark/automl/VerifyFindBestModel.scala

```diff
@@ -98,6 +98,6 @@ class VerifyFindBestModel extends EstimatorFuzzing[FindBestModel]{
     val randomForestClassifier = createRF.setLabelCol(mockLabelColumn)
     val model = randomForestClassifier.fit(createMockDataset)
     new FindBestModel()
-      .setModels(Array(model, model))
+      .setModels(Array(model.asInstanceOf[Transformer], model))
```

Comment:
this now needed?

Reply:
Emm yes.. Otherwise it has `cannot resolve overloaded method` error